include/seqan3/core/bit_manipulation.hpp

```diff
@@ -51,9 +49,9 @@ constexpr auto sizeof_bits = min_viable_uint_v<CHAR_BIT * sizeof(type_t)>;
  *
  * \sa https://graphics.stanford.edu/~seander/bithacks.html#DetermineIfPowerOf2
  */
-constexpr bool is_power_of_two(size_t const n)
+SEQAN3_DEPRECATED_310 constexpr bool is_power_of_two(size_t const n)
```

Comment:
for all those things deprecated, the doxygen documentation should be adapted and point to the respective std alternative

Reply:
good idea :)

I didn't do that, because they are detail, but it is a good idea :)